### ISSUE TITLE
Add "Skip to Main Content" button

### ISSUE THREAD

**Issue by TheEvilSkeleton** [Vanilla-OS/website#64]
Pressing Tab immediately jumps to the navigation bar, which requires extra steps for people who use screen readers (i.e. it's inaccessible). Instead, there should be a hidden "Skip to Content" button that appears when the user presses Tab as soon as the page loads.

You can use my website as an example: https://theevilskeleton.gitlab.io. When the page starts, press Tab, followed by Enter/Return. It will scroll to the content and skip the navigation.

Here's the implementation (CSS): https://gitlab.com/TheEvilSkeleton/theevilskeleton.gitlab.io/-/blob/df9122e4835802065a3ae1c00bbe2de6e19431cb/src/styles/styles.css#L258-268
